### DISCUSSION TITLE
Is this project still maintained?

### DISCUSSION THREAD

**Discussion by danimatuko** [dam9000/kickstart-modular.nvim#31] [Q&A] [answered]
I find it easier to start with this project instead of the original. However, it’s been a few months since the last commit here, while the original project has been updated much more frequently.

Can I trust thus will be up to date with the original kickstart?

**Comment by ctarx** [comment]
As far as I'm concerned, it looks like it's still being maintained—just look at the recent commit history. My setup is also based on this fork.

**Comment by vlanx** [comment]
I'm wondering how you come to the conclusion of how there hasn't been a commit in months? The `master` branch has been regularly updated, plus the owner of this fork is a regular contributor of the main `kickstart.nvim` project. I don't see a reason to not trust this fork.

**Comment by dam9000** [comment]
Yes the project is actively maintained.
As @dot-1q already noted I also don't understand how you concluded that "it’s been a few months since the last commit"

**Reply by danimatuko** [reply, marked as answer]
Thank you all for the answers. I mistakenly pressed This` branch is [97 commits ahead of] nvim-lua/kickstart.nvim:master `instead of the `commits` button.

Thank you : )
